src/Stratis.Bitcoin.Features.SmartContracts/ReflectionExecutor/Controllers/SmartContractsController.cs

```diff
@@ -113,6 +129,19 @@ public IActionResult GetBalance([FromQuery]string address)
             return Json(balance);
         }
 
+        /// <summary>
+        /// Gets a single piece of smart contract data, which was stored using the
+        /// SmartContract.PersistedData property as a key–value pair.
```

Comment:
Should be `SmartContract.PersistentState`. Though I think this line can be removed altogether because the information is repeated below.

Reply:
Quite like the specific tie-in with the code even though we elaborate on key-value pairs below. Will fix the code typo up of course!